cores/rp2040/_freertos.h

```diff
@@ -43,10 +43,10 @@ extern "C" {
     extern SemaphoreHandle_t _freertos_recursive_mutex_create() __attribute__((weak));
 
     extern void __freertos_mutex_take(SemaphoreHandle_t mtx) __attribute__((weak));
-    extern void __freertos_mutex_take_from_isr(SemaphoreHandle_t mtx) __attribute__((weak));
+    extern int __freertos_mutex_take_from_isr(SemaphoreHandle_t mtx) __attribute__((weak));
```

Comment:
This call also needs to include a pointer param to return the portYield bool returned from the takeFromISR call.

Reply:
Done